lib/src/Pos/Block/Worker.hs

```diff
@@ -117,7 +118,11 @@ blockCreator (slotId@SlotId {..}) sendActions = do
         jsonLog $ jlCreatedBlock (Left createdBlk)
 
     -- Then we get leaders for current epoch.
-    leadersMaybe <- getLeaders siEpoch
+    -- Note: we are using non-blocking version here.  If we known
```

Comment:
This is very obsolete comment, which was removed in master. So it shouldn't be here. But it CI passes and someone else approves without any requested changes, I don't mind merging it.